tests/test_worker.py

```diff
@@ -368,35 +370,44 @@ async def test_schedule(self, mock_time: MagicMock) -> None:
         self.assertEqual(job.result, 1)
 
     @mock.patch("saq.worker.logger")
-    @mock.patch("saq.utils.time")
-    async def test_cron(self, mock_time: MagicMock, mock_logger: MagicMock) -> None:
+    async def test_cron(self, mock_logger: MagicMock) -> None:
         with self.assertRaises(ValueError):
             Worker(
                 self.queue,
                 functions=functions,
-                cron_jobs=[CronJob(cron, cron="x")],
+                cron_jobs=[CronJob(sleeper, cron="x")],
```

Comment:
why is this function changed?

Reply:
See line 404, one of the checks is if the job is running which requires it to take some time.